src/logic/hooks/useEstimateSafeCreationGas.tsx

```diff
@@ -41,6 +46,8 @@ const estimateGas = async (
     gasEstimation,
     gasCostFormatted,
     gasLimit: gasEstimation,
+    gasMaxPrioFee: maxPrioFeePerGas,
+    gasMaxPrioFeeFormatted: formatAmount(maxPrioFeePerGas.toString()),
```

Comment:
Is the formatted max prio fee used for anything?

Reply:
It doesn't seem so, but why should we remove it?